rust/src/gildedrose.rs

```diff
@@ -88,12 +86,86 @@ impl GildedRose {
 mod tests {
     use super::{GildedRose, Item};
 
+    pub fn days_loop(rose: &mut GildedRose, mut days: i32) {
+        while days > 0 {
+            rose.update_quality();
+            days -= 1;
+        }
+    }
+
     #[test]
-    pub fn foo() {
-        let items = vec![Item::new("foo", 0, 0)];
-        let mut rose = GildedRose::new(items);
-        rose.update_quality();
+    pub fn default_item() {
```

Comment:
As above this function doesnt need to be private

Reply:
yes, algo I forgot to change the functions name. That was for the first try I did